scripts/install-third-party-renderers.sh

```diff
@@ -87,7 +87,7 @@ fi
 
 pushd $FILAMENT_DIR
 
-git fetch --depth=1 origin
+#git fetch --depth=1 origin
```

Comment:
Remove or add comment?

Reply:
Removed by accident (couldn't `git fetch` from the commuter ferry 🙃)

Reply:
Fixed